x-pack/legacy/plugins/maps/public/layers/sources/es_search_source/create_source_editor.js

```diff
@@ -46,13 +46,10 @@ export class CreateSourceEditor extends Component {
     onSourceConfigChange: PropTypes.func.isRequired,
   };
 
-  static defaultProps = {
-    filterByMapBounds: DEFAULT_FILTER_BY_MAP_BOUNDS,
-  };
-
   state = {
     isLoadingIndexPattern: false,
     noGeoIndexPatternsExist: false,
+    filterByMapBounds: DEFAULT_FILTER_BY_MAP_BOUNDS,
```

Comment:
This line is not needed. `RESET_INDEX_PATTERN_STATE` already contains a default value for `filterByMapBounds`